examples/example-tls/README.md

```diff
@@ -35,65 +38,27 @@ USAGE: HelloWorldClientTls host port trustCertCollectionFilePath [clientCertChai
   Note: clientCertChainFilePath and clientPrivateKeyFilePath are only needed if mutual auth is desired.
 ```
 
-#### Generating self-signed certificates for use with grpc
-
-You can use the following script to generate self-signed certificates for grpc-java including the hello world with TLS examples:
+We provide a set of self-signed client/server keys and certificates to run the hello world with TLS examples on 
+`localhost`. They can be found in `src/main/resources/certs`. You will need to generate/supply your own 
+client/server keys and certificates with Subject Alternative Names in other domains.
```

Comment:
What about "You can provide your own keys and certificates but be aware that the system may require a valid Subject Alternative Names."

Reply:
Done.